modules/cTab_settings/functions/CTAB_SET/script_component.hpp

```diff
@@ -0,0 +1,2 @@
+#define COMPONENT CAPZONE
```

Comment:
Incorrect component name, no functions are used so functions can be taken out couldn't they?

Reply:
Yeah, easy enough to add in when it's needed.